bin/docker_start

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+require 'uri'
+
+# check that postgres is up
+pg_uri = URI.parse(ENV['DATABASE_URL'])
+`dockerize -wait tcp://#{pg_uri.host}:#{pg_uri.port || 5432} -timeout 60s`
+`bash -c 'if [ "$AUTOMIGRATE" == "1" ]; then bundle exec rails db:create db:migrate; fi'`
```

Comment:
You can simplify this a bit by doing the conditional in Ruby rather than Bash.

Reply:
switched